src/site/xdoc/server/config-cassandra.xml

```diff
@@ -114,13 +114,13 @@
         <dt><strong>mailbox.blob.part.size</strong></dt>
         <dd>Optional. Defaults to 102400 (100KB).<br/> Controls the size of blob parts used to store messages.</dd>
         <dt><strong>migration.v1.v2.on.the.fly</strong></dt>
-        <dd>Optional. Defaults to false.<br/> Controls wether v1 to v2 migration should be run on the fly.</dd>
+        <dd>Only available on tag cassandra_migration_v1_to_v2. Optional. Defaults to false.<br/> Controls wether v1 to v2 migration should be run on the fly.</dd>
```

Comment:
Found: move them in the migration section, give details about each migration.

![capture d ecran de 2017-08-18 09-43-14](https://user-images.githubusercontent.com/6928740/29442125-e7a029d8-83f9-11e7-908e-adc888363f90.png)